src/transformers/__init__.py

```diff
@@ -205,6 +207,12 @@
     "models.gpt2": ["GPT2_PRETRAINED_CONFIG_ARCHIVE_MAP", "GPT2Config", "GPT2Tokenizer"],
     "models.gpt_neo": ["GPT_NEO_PRETRAINED_CONFIG_ARCHIVE_MAP", "GPTNeoConfig"],
     "models.gptj": ["GPTJ_PRETRAINED_CONFIG_ARCHIVE_MAP", "GPTJConfig"],
+    "models.greaselm": [
+        "GREASELM_PRETRAINED_CONFIG_ARCHIVE_MAP",
+        "GreaseLMConfig",
+        "GreaseLMFeatureExtractor",
+        "GreaseLMProcessor",
```

Comment:
The processor seems to depend on scatter and sparse, hence it shouldn't be here but under its own dependency checks